src/compiler/compile/css/Selector.ts

```diff
@@ -163,6 +164,17 @@ export default class Selector {
 			}
 		}
 	}
+	validate_invalid_combinator_without_selector(component: Component) {
+		for (let i = 0; i < this.blocks.length; i++) {
+			const block = this.blocks[i];
+			if (block.combinator && block.selectors.length === 0) {
+				component.error(this.node, compiler_errors.css_invalid_selector(component.source.slice(this.node.start, this.node.end)));
+			}
```

Comment:
I think added tests don't go inside the IF statement.
Maybe you need to add tests like the below case.

```svelte
<span><p>foo</p></span>

<style>
	span > {
		color: red;
	}
</style>
```

Reply:
added